server.js

```diff
@@ -24,9 +27,130 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
+const Song = mongoose.model('Song', {
+  id: Number,
+  trackName: String,
+  artistName: String,
+  genre: String,
+  bpm: Number,
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number
+});
+
+const Movie = mongoose.model('Movie', {
+  show_id: Number,
+  title: String,
+  director: String,
+  cast: String,
+  country: String,
+  date_added: String,
+  release_year: Number,
+  rating: String,
+  duration: String,
+  listed_in: String,
+  description: String,
+  type: String
+});
+
+if (process.env.RESET_DB) {
+  const resetDataBase = async () => {
+    await Song.deleteMany();
+    topMusicData.forEach((singleSong) => {
+      const newSong = new Song(singleSong);
+      newSong.save();
+    });
+    netflixData.forEach((singleMovie) => {
+      const newMovie = new Movie(singleMovie);
+      newMovie.save();
+    });
+  };
```

Comment:
Cool that you used two datasets for this project. Nice personal touch! I'm wondering how come only the song-collection get deleted when reseting database. Why doesn't the movies-collection keep adding on?

Reply:
Haha, nice catch! I suppose it would create a lot of copies of movies if I ran it several times with RESET_DB=true. Will try it out and fix.